devops/pack-templates.cmd

```diff
@@ -0,0 +1,6 @@
+call copy-template.cmd LogoFX.Templates.WPF --use-common
+cd ..\..\..\devops
```

Comment:
Return to 'devops' in copy-template.cmd

Reply:
It doesn't work locally. try to run the scenarios and the script and you'll see